kit/react-select/index.ts

```diff
@@ -17,4 +17,5 @@ export const reactSelect = elementFactory(Select),
     reactCreatableSelect = elementFactory(Creatable),
     reactAsyncSelect = elementFactory(AsyncSelect),
     reactAsyncCreatableSelect = elementFactory(AsyncCreatable),
-    reactWindowedSelect = elementFactory(WindowedSelect);
+    // Typed as any due to issue with react-windowed-select types
```

Comment:
how does this manifest?  Will typescript complain?  if so, comment not adding anything

Reply:
Here's the TS error without casting `WindowedSelect`

<img width="1356" alt="Screenshot 2024-07-01 at 12 14 35 PM" src="https://github.com/xh/hoist-react/assets/88904581/892515ca-fbcf-46d9-9229-2d5ad2c733ac">